### PR TITLE
Avatar animations

### PR DESCRIPTION
**Pull request by behowell**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Add animations for the Avatar's Active<->Inactive state transitions

### CODE REVIEW

File: packages/react-avatar/src/components/Avatar/Avatar.scss [hunk 1]
```diff
@@ -267,7 +267,12 @@ $default-inactiveScaleFactor: 0.8;
 }
 
 // Activity modifiers
-._active {
+._active,
+._inactive {
+  // Work-around pixel snapping jankiness at the end of the animation if the scale factor is 1.0
+  transform: scale(0.9999);
```
Comment (behowell) [inline packages/react-avatar/src/components/Avatar/Avatar.scss:273]: Any opinions on this workaround? Without it the avatar jumps at the end of the scale animation.

Comment (dzearing) [reply packages/react-avatar/src/components/Avatar/Avatar.scss:273]: I'd have to pull to see what you are;

**Comment by dzearing** [comment]
It would be good to add this to the demo site, or add to codesandbox templates. Otherwise can't see the changes you're making.

File: packages/react-avatar/src/components/Avatar/_tokens.scss [hunk 2]
```diff
@@ -158,3 +158,41 @@ $shadow-16-darker: 0px 0px 4px $color-default-shadowAmbientDarker, 0px 6px 16px
...
+$anim-ultraFast: $anim-duration50;
+$anim-faster: $anim-duration100;
+$anim-fast: $anim-duration150;
+$anim-normal: $anim-duration200;
+$anim-slow: $anim-duration300;
+$anim-slower: $anim-duration400;
+$anim-ultraSlow: $anim-duration500;
+
+// Animation curves
+$anim-decelerateMax: cubic-bezier(0.00,0.00,0.00,1.00);
```
Comment (dzearing) [inline packages/react-avatar/src/components/Avatar/_tokens.scss:180]: Seems like we'd probably want to move this to @fluentui/theme.

**Review by dzearing — APPROVED** [review]

File: packages/react-avatar/src/components/Avatar/_tokens.scss (hunk 2 above)
Comment (behowell) [reply packages/react-avatar/src/components/Avatar/_tokens.scss:180]: Yeah eventually all of these should come from design tokens via the theme. I didn't want that work to block me here, so I just added them here for now until the rest of that is figured out.

**Comment by behowell** [comment]
> It would be good to add this to the demo site, or add to codesandbox templates. Otherwise can't see the changes you're making.

Are there instructions on how to do that?

**Pull request merged**